### ISSUE TITLE
Install Lefthook using a package manager in Windows

### ISSUE THREAD

**Issue by SAGARSURI** [evilmartians/lefthook#136]
Is there a way to install lefthook on Windows using `chocolatey` package manager or `winget`? I want to write a bash script to automate this process of installing lefthook on my machine and other teammates.

**Comment by Arkweid** [comment]
@SAGARSURI Hi, you could add lefthook in any package manager you want, like here for python: https://github.com/life4/lefthook

**Comment by SAGARSURI** [comment]
Hey @Arkweid , Is it possible to add the `.exe` file in the asset of the release notes? I am planning to open up a PR for the `winget` package manager. They expect to mention the type of installer while submitting the PR.

**Comment by Arkweid** [comment]
Oke, I will add unzipped file in next release.

**Comment by SAGARSURI** [comment]
Thank you so much! @Arkweid

**Comment by SAGARSURI** [comment]
I have added this to my JIRA ticket and will resume work once this is done. 😄

**Comment by ahuglajbclajep** [comment]
On Windows 1803 or later, you can use Unix `curl` with `curl` (in CMD) or `curl.exe` (in PowerShell).
So when unzipped files are added to the release, you can install `lefthook` with the same commands in both Windows and Unix (although you have to change the download URL and set the `PATH`) 🎉

**Comment by bbodenmiller** [comment]
It does not appear `exe` ever started getting published. @Envek could this be added for next release?

**Comment by SAGARSURI** [comment]
Thank you so much @Envek

**Comment by aminya** [comment]
This is counterintuitive. You should probably use a better package manager. For example, Chocolatey is able to unzip files without any issues.

**Comment by SAGARSURI** [comment]
Why should we use Chocolatey when Microsoft provide a default package manager i.e `winget`. It provides better security compared to other 3rd party package manager.

**Comment by aminya** [comment]
> Why should we use Chocolatey when Microsoft provide a default package manager i.e `winget`. It provides better security compared to other 3rd party package manager.

I'm very sorry to see this attitude toward community-driven open-source projects. A 3rd party project is not insecure by default, and Microsoft projects are **not** any more secure. Instead, I suggest you decide based on the actual technical differences. For example, if a package manager lacks its basics such as installing zip files or if it allows decentralized packages.

_P.S: since this is offtopic, I will not continue this discussion after this comment._

**Comment by Envek** [comment]
We want to support as many installations methods as possible (however, using as least effort as possible). So, let's add support to both (pull requests are welcome)

**Comment by sheldonhull** [comment]
Chocolatey is the defacto (winget is newer), but for Go packages I'd suggest Scoop. Goreleaser has built in configuration to support Scoop. I've written chocolatey packages and it's not as straight forward. 
Between Scoop, `go install`, and npm install you've covered a nice set of install options.

https://goreleaser.com/customization/scoop/

**Comment by nikoheikkila** [comment]
I would also love to have this installable via Scoop. I have some experience with GoReleaser so let me know if help is needed.

**Comment by sitiom** [comment]
> I would also love to have this installable via Scoop. I have some experience with GoReleaser so let me know if help is needed.

Now available in Scoop via the Main bucket:
- https://github.com/ScoopInstaller/Main/pull/4967

GoReleaser is no longer needed as the bucket auto-updates the manifest itself.

As for Winget, I have opened a PR for it: https://github.com/microsoft/winget-pkgs/pull/112810. Once that's merged, the Winget Releaser action should be used to keep it up-to-date:
- #526

**Comment by mrexox** [comment]
@sitiom , thank you a lot! ❤️🔥
